### PR TITLE
Update community supported README: legal disclaimer

### PR DESCRIPTION
**Pull request by cwarnermm**
Added legal disclaimer to README, as requested by Product.

### CODE REVIEW

**Review by hanzei — DISMISSED** [review]
Blocking until the discussion in https://community.mattermost.com/private-core/pl/51hbdxfn7tbbuboys4siw8peea has been resolved

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+# :warning: Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims any and all liability for integrations, including Third Party Integrations and Mattermost Integrations. All integrations are provided "AS IS", and may be used at your own risk. Integrations may be modified or discontinued at any time.**
```
Comment (cwarnermm) [inline README.md:3]:
```suggestion
# Disclaimer

**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```

**Comment by cwarnermm** [comment]
@hanzei - Legal's updated guidance and language for the legal disclaimer for community-supported integrations has been applied to this PR.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Disclaimer
+
+**This repository is community supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including Third Party Integrations and Mattermost Integrations. Integrations may be modified or discontinued at any time.**
```
Comment (mickmister) [inline README.md:3]: 0/5 on if this is correct. Maybe not worth changing if legal needs to revisit the wording

```suggestion
**This repository is community-supported and not maintained by Mattermost. Mattermost disclaims liability for integrations, including third-party integrations and Mattermost integrations. Integrations may be modified or discontinued at any time.**
```

> Mattermost disclaims liability for ... Mattermost integrations

Is this true?

Comment (cwarnermm) [reply README.md:3]: This wording has been provided from Legal and should not be modified.

**Review by mickmister — APPROVED** [review]

**Comment by cwarnermm** [comment]
@hanzei - This PR is ready for your final approval.

**Review by Willyfrog — APPROVED** [review]

**Pull request merged**